iosxe_static_route.tf

```diff
@@ -0,0 +1,25 @@
+locals {
+  static_routes = flatten([
+    for device in local.devices : [
+      for static_route in try(local.device_config[device.name].routing.static_routes, []) : {
+        device_name = device.name
+        prefix      = static_route.prefix
```

Comment:
We need a `try()` construct to consistently apply default values (see some of the existing tf files).